Gemfile

```diff
@@ -5,14 +5,16 @@ ruby '2.6.5'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.0'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
```

Comment:
Per our Slack chat yesterday, let's keep sqlite3 for dev and test environments. We'll use mysql in production only.